ports/renesas-ra/mpbthciport.c

```diff
@@ -37,9 +39,6 @@
 
 #if MICROPY_PY_BLUETOOTH
 
-#define debug_printf(...) // mp_printf(&mp_plat_print, "mpbthciport.c: " __VA_ARGS__)
-#define error_printf(...) mp_printf(&mp_plat_print, "mpbthciport.c: " __VA_ARGS__)
```

Comment:
These debug macros have lost the context of what file the log line is coming from.

Reply:
See  do_print_line()